index.bs

```diff
@@ -259,6 +261,38 @@ the user agent SHOULD run these steps:
 2. If |video| is {{pictureInPictureElement}}, run the <a>exit
     Picture-in-Picture algorithm</a>.
 
+## Auto Picture-in-Picture ## {#auto-pip}
+
+Some pages may want to automatically enter and leave Picture-in-Picture for
+a video element; for example, video meetings web apps would benefit from
+some automatic Picture-in-Picture behavior when user switches back and forth
+between web app and other applications/tabs.
+To support these use cases, a new {{autoPictureInPicture}} attribute is
+added to the list of content attributes for video elements.
+
+A corresponding {{autoPictureInPicture}} IDL attribute which reflects the
+value of element’s {{autoPictureInPicture}} content attribute is added to
+the {{HTMLVideoElement}} interface. The {{autoPictureInPicture}} IDL attribute
+MUST <a>reflect</a> the content attribute of the same name.
+
+The <dfn>autoPictureInPictureElement</dfn> is the video element whose
+{{autoPictureInPicture}} attribute was set most recently.
+
+When the visibility state of a <a>top-level browsing context</a> value
+switches from "hidden" to "visible", the user agent MAY run these steps:
+
+1. If <a>autoPictureInPictureElement</a> is null, abort these steps.
+2. Let |video| be <a>autoPictureInPictureElement</a>.
+3. Let |userActivationRequired| be `false`.
+4. Run the <a>request Picture-in-Picture algorithm</a> with |video| and
+    |userActivationRequired|.
+
+When the visibility state of a <a>top-level browsing context</a> value
+switches from "visible" to "hidden", the user agent MAY run these steps:
+
+1. If <a>autoPictureInPictureElement</a> is null, abort these steps.
+2. Run the <a>exit Picture-in-Picture algorithm</a>.
```

Comment:
We should also check that the `pictureInPictureElement` is `autoPictureInPictureElement`

Reply:
You're right. Thanks!